python/tvm/driver/tvmc/compiler.py

```diff
@@ -42,6 +42,13 @@ def add_compile_parser(subparsers):
 
     parser = subparsers.add_parser("compile", help="compile a model.")
     parser.set_defaults(func=drive_compile)
+    parser.add_argument(
+        "--config",
+        action="append",
+        metavar=("name=value"),
+        help="configurations to be used at compile time. A subset of options provided "
+        "by TVM are supported. e.g. 'relay.backend.use_auto_scheduler=0'",
```

Comment:
> I'm wondering if it would make sense to enhance the help message a bit more so users don't try to do something like:

Fixed.

> 
> I also see duplicated and even conflicting flags don't generate any error or warning. Should we treat them too? Like:
> 

I think most tools won't complain if you provide repeated configs. In this case, similar to what most tools e.g. _docker, bash_, will just assume the latest (considering parsing is done left to right) value is the one to be used.

note: In some other cases, like in the `--target`, I'm validating duplicates because I need to translate that plain string to various internal APIs.

Reply:
OK. Yeah I was indeed thinking of how we deal with `--target` ;)